recruitments/serializers.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework import serializers
+
+from employees.models import Employee
+from recruitments.models import Recruits, Referrals
+
+
+class RecruitsSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = Recruits
+        fields = '__all__'
+
+    def create(self, validated_data):
+        recruits = Recruits.objects.create(**validated_data)
+        if self.initial_data.get('referrers'):
+            emp = Employee.objects.get(id=self.initial_data.get('referrers'))
+            if emp:
+                Referrals.objects.create(referer=emp, recruit=recruits)
```

Comment:
I guess we can do it in either way like using the direct refferrer id with Referral Serializer to validate & save it